### PR TITLE
[mega branch] main spring launch event PR

### PR DESCRIPTION
**Pull request by mirnawong1**
This PR and branch serves as the main (mega) branch the docs team will use to merge their changes into for the spring launch showcase event. 

Some of the changes inlcude:
- lifecycle status updates
- product-related updates
- spring launch joint release notes

### CODE REVIEW

**Comment by mirnawong1** [comment]
hey @graciegoheen @jtcohen6 wanted to give you an fyi that this is the mega pr that also removes any 'preview/beta' mention for unit tests/dbt mesh, and all the other products that are announced as gaon tues.

File: website/docs/docs/dbt-versions/release-notes.md [hunk 1]
```diff
@@ -18,8 +18,65 @@ Release notes are grouped by month. For customers using dbt Virtual Private Clou
...
+  - [Column-level lineage](/docs/collaborate/column-level-lineage) 
+  - [Model performance](/docs/collaborate/model-performance) 
+  - [Project recommendations](/docs/collaborate/project-recommendations) 
+
+  </expandable>
+
+- **New:** Native support for Microsoft Fabric in dbt Cloud is now GA. This feature is powered by the [dbt-fabric](https://github.com/Microsoft/dbt-fabric) adapter. To learn more, refer to [Connect Microsoft Fabric](/docs/cloud/connect-data-platform/connect-microsoft-fabric) and [Microsoft Fabric DWH configurations](/reference/resource-configs/fabric-configs). There's also a [quickstart guide](https://docs.getdbt.com/guides/microsoft-fabric?step=1) to help you get started. 
+
+- **New:** [dbt Mesh](ttps://docs.getdbt.com/best-practices/how-we-mesh/mesh-1-) is now GA to dbt Cloud Enterprise users. dbt Mesh is a framework that helps organizations scale their teams and data assets effectively. It promotes governance best practices and breaks large projects into manageable sections. Get started with dbt Mesh by reading the [dbt Mesh quickstart guide](https://docs.getdbt.com/guides/mesh-qs?step=1).
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes.md]:
```suggestion
- **New:** [dbt Mesh](https://docs.getdbt.com/best-practices/how-we-mesh/mesh-1-) is now GA to dbt Cloud Enterprise users. dbt Mesh is a framework that helps organizations scale their teams and data assets effectively. It promotes governance best practices and breaks large projects into manageable sections. Get started with dbt Mesh by reading the [dbt Mesh quickstart guide](https://docs.getdbt.com/guides/mesh-qs?step=1).
```

File: website/docs/docs/dbt-versions/release-notes.md [hunk 2]
```diff
@@ -18,8 +18,65 @@ Release notes are grouped by month. For customers using dbt Virtual Private Clou
...
+  - Documenting your models.
+  - Scheduling a job to run.
+
+  </expandable>
+
+- **New:** [dbt Assist](/docs/cloud/dbt-assist) is a powerful AI feature, available in beta, helping you generate documentation, metrics, semantic models, and tests, saving you time and helping you deliver data that works. Available for dbt Cloud Enterprise accounts in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).
+
+- **New:** The VS Code extension [Power user for dbt Core and dbt Cloud](https://docs.myaltimate.com/arch/beta/) is now available in beta for [dbt Cloud CLI](/docs/cloud/cloud-cli-installation#vs-code-extension) users. The extension accelerates dbt and SQL development and includes features such as generating models from your source definitions or SQL, updating dbt model using natural language, and [more](https://docs.myaltimate.com/)!
+
+- **New:** MetricFlow enables you to now add [metrics as dimensions](/docs/build/ref-metrics-in-filters) to your metric filters to create more complex metrics and gain more insights. Available for all users.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes.md]:
```suggestion
- **New:** MetricFlow enables you to now add metrics as dimensions to your metric filters to create more complex metrics and gain more insights. Available for all users.
```

File: website/docs/docs/dbt-versions/release-notes.md [hunk 3]
```diff
@@ -18,8 +18,65 @@ Release notes are grouped by month. For customers using dbt Virtual Private Clou
...
+  - [Column-level lineage](/docs/collaborate/column-level-lineage) 
+  - [Model performance](/docs/collaborate/model-performance) 
+  - [Project recommendations](/docs/collaborate/project-recommendations) 
+
+  </expandable>
+
+- **New:** Native support for Microsoft Fabric in dbt Cloud is now GA. This feature is powered by the [dbt-fabric](https://github.com/Microsoft/dbt-fabric) adapter. To learn more, refer to [Connect Microsoft Fabric](/docs/cloud/connect-data-platform/connect-microsoft-fabric) and [Microsoft Fabric DWH configurations](/reference/resource-configs/fabric-configs). There's also a [quickstart guide](https://docs.getdbt.com/guides/microsoft-fabric?step=1) to help you get started. 
+
+- **New:** [dbt Mesh](https://docs.getdbt.com/best-practices/how-we-mesh/mesh-1) is now GA to dbt Cloud Enterprise users. dbt Mesh is a framework that helps organizations scale their teams and data assets effectively. It promotes governance best practices and breaks large projects into manageable sections. Get started with dbt Mesh by reading the [dbt Mesh quickstart guide](https://docs.getdbt.com/guides/mesh-qs?step=1).
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes.md]: Fixing to fix failing test
```suggestion
- **New:** dbt Mesh is now GA to dbt Cloud Enterprise users. dbt Mesh is a framework that helps organizations scale their teams and data assets effectively. It promotes governance best practices and breaks large projects into manageable sections. Get started with dbt Mesh by reading the [dbt Mesh quickstart guide](https://docs.getdbt.com/guides/mesh-qs?step=1).
```

File: website/docs/docs/dbt-versions/release-notes.md [hunk 4]
```diff
@@ -18,8 +18,69 @@ Release notes are grouped by month. For customers using dbt Virtual Private Clou
...
+  - Turning a sample query into a model in your dbt project. A model in dbt is a SELECT statement.
+  - Adding tests to your models.
+  - Documenting your models.
+  - Scheduling a job to run.
+
+  </expandable>
+
+- **New:** MetricFlow enables you to now add metrics as dimensions to your metric filters to create more complex metrics and gain more insights. Available for all dbt Semantic Layer users.
+
+- **New:** [Staging environment](/docs/deploy/deploy-environments#staging-environment) is now GA. Use staging environments to grant developers access to deployment workflows and tools while controlling access to production data. Available to dbt Cloud users.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes.md]:
```suggestion
- **New:** [Staging environment](/docs/deploy/deploy-environments#staging-environment) is now GA. Use staging environments to grant developers access to deployment workflows and tools while controlling access to production data. Available to all users.
```

**Review by runleonarun — APPROVED** [review]
let's ship this puppy 🐶

**Pull request merged**